core/src/main/java/org/hisp/dhis/android/core/domain/metadata/MetadataCall.kt

```diff
@@ -130,15 +130,15 @@ internal class MetadataCall @Inject constructor(
         useCaseDownloader.downloadMetadata().blockingAwait()
         emit(progressManager.increaseProgress(StockUseCase::class.java, false))
 
-        constantModuleDownloader.downloadMetadata().blockingGet()
+        runBlocking { constantModuleDownloader.downloadMetadata() }
```

Comment:
The `runBlocking` is not needed. This line is inside a flow, so you can call suspend functions directly.